drivers/mcp2515/mcp2515_defines.h

```diff
@@ -300,25 +300,15 @@ extern "C" {
 #define MCP2515_RXB0CTRL_BUKT1                  0x02
 #define MCP2515_RXB0CTRL_BUKT                   0x04
 #define MCP2515_RXB0CTRL_RXRTR                  0x08
-#define MCP2515_RXB0CTRL_RXM0                   0x20
-#define MCP2515_RXB0CTRL_RXM1                   0x40
-#define MCP2515_RXB0CTRL_MODE_RECV_STD_OR_EXT   0x00
-#define MCP2515_RXB0CTRL_MODE_RECV_STD          MCP2515_RXB0CTRL_RXM0
-#define MCP2515_RXB0CTRL_MODE_RECV_EXT          MCP2515_RXB0CTRL_RXM1
-#define MCP2515_RXB0CTRL_MODE_RECV_ALL          (MCP2515_RXB0CTRL_RXM1 | \
-                                                 MCP2515_RXB0CTRL_RXM0)
+#define MCP2515_RXB0CTRL_MODE_RECV_FILTER       0x00
+#define MCP2515_RXB0CTRL_MODE_RECV_ALL          0x60
 
 #define MCP2515_RXB1CTRL_FILHIT0                0x01
 #define MCP2515_RXB1CTRL_FILHIT1                0x02
 #define MCP2515_RXB1CTRL_FILHIT2                0x04
 #define MCP2515_RXB1CTRL_RXRTR                  0x08
-#define MCP2515_RXB1CTRL_RXM0                   0x20
-#define MCP2515_RXB1CTRL_RXM1                   0x40
-#define MCP2515_RXB1CTRL_MODE_RECV_STD_OR_EXT   0x00
-#define MCP2515_RXB1CTRL_MODE_RECV_STD          MCP2515_RXB1CTRL_RXM0
-#define MCP2515_RXB1CTRL_MODE_RECV_EXT          MCP2515_RXB1CTRL_RXM1
-#define MCP2515_RXB1CTRL_MODE_RECV_ALL          (MCP2515_RXB1CTRL_RXM1 | \
-                                                 MCP2515_RXB1CTRL_RXM0)
+#define MCP2515_RXB1CTRL_MODE_RECV_FILTER       0x00
+#define MCP2515_RXB1CTRL_MODE_RECV_ALL          0x60
```

Comment:
I agree that dropping the unused defines is unnecessary since they just encode information about the hardware and could become useful in the future.